docs/docs/reference/migration.md

```diff
@@ -3,51 +3,81 @@
 This document describes breaking changes and migrations between Constellation releases.
 Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to automatically update an old config file to a new format.
 
+## Migrations to v2.19.1
+
+### Azure
+
+* During the upgrade, security rules are migrated and the old ones need to be cleaned up manually by the user. The below script shows how to delete them through the Azure CLI:
+
+```bash
+#!/usr/bin/env bash
+name="<insert>"  # the name provided in the config
+uid="<insert>"    # the cluster id can be retrieved via `yq '.infrastructure.uid' constellation-state.yaml`
+resource_group="<insert>" # the RG can be retrieved via `yq '.provider.azure.resourceGroup' constellation-conf.yaml`
+
+rules=(
+  "kubernetes"
+  "bootstrapper"
+  "verify"
+  "recovery"
+  "join"
+  "debugd"
+  "konnectivity"
+)
+
+for rule in "${rules[@]}"; do
+  echo "Deleting rule: ${rule}"
+  az network nsg rule delete \
+    --resource-group "${resource_group}" \
+    --nsg-name "${name}-${uid}" \
+    --name "${rule}"
+done
+
+echo "All specified rules have been deleted."
+```
 
 ## Migrations to v2.19.0
 
 ### Azure
 
-* To allow seamless upgrades on Azure when Kubernetes services of type `LoadBalancer` are deployed, the target  
+* To allow seamless upgrades on Azure when Kubernetes services of type `LoadBalancer` are deployed, the target
  load balancer in which the `cloud-controller-manager` creates load balancing rules was changed. Instead of using the load balancer
  created and maintained by the CLI's Terraform code, the `cloud-controller-manager` now creates its own load balancer in Azure.
  If your Constellation has services of type `LoadBalancer`, please remove them before the upgrade and re-apply them
- afterward. 
-
+ afterward.
 
 ## Migrating from Azure's service principal authentication to managed identity authentication (during the upgrade to Constellation v2.8.0)
 
-- The `provider.azure.appClientID` and `provider.azure.appClientSecret` fields are no longer supported and should be removed.
-- To keep using an existing UAMI, add the `Owner` permission with the scope of your `resourceGroup`.
-- Otherwise, simply [create new Constellation IAM credentials](../workflows/config.md#creating-an-iam-configuration) and use the created UAMI.
-- To migrate the authentication for an existing cluster on Azure to an UAMI with the necessary permissions:
+* The `provider.azure.appClientID` and `provider.azure.appClientSecret` fields are no longer supported and should be removed.
```

Comment:
I don't think we currently enforce any rules regarding `*` vs `-`. They seem to be interchangeable. Did your linter change this, or do you have a reason for this preference? I know that docs currently favor `*` (i.e. are used more). I think we can revert this for now to not have the last commit that changed the line updated and revisit this discussion later.

Reply:
We agreed on markdownlint, which (unfortunately) picks the character of the topmost list item on the page. So if you wanted to keep the diff small but still use the linter, you would replace the first item with `-` in this case.